src/App.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+import Signup from './components/auth/signup/Signup';
 
 function App() {
   return (
-    <div className="App">
-      <h1 className="text-danger">Welcome To Anonymous Team Feedback</h1>
-    </div>
+    <Router>
+      <Route exact path="/signup" component={Signup} />
```

Comment:
I think you should wrap the `Routes` with `Switch` HOC so that when other routes are added there wouldn't be clashes in the route page rendering

Reply:
Okay